bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -259,6 +259,30 @@ public String getLang() {
         }
     }
 
+    @Override
+    public String getDirection() {
+        if (isLangRTL()) {
```

Comment:
Can you create something in ComponentUtils for this, 

`ComponentUtils.getLanguageDirection` and move the entire logic there